### ISSUE TITLE
Sample WAV not playable (in some players)

### ISSUE THREAD

**Issue by MasterJames** [mramshaw/Alexa-Stuff#1]
I'm not sure why I've never seen anything like it after 30+ years in audio.
https://github.com/mramshaw/Alexa-Stuff/blob/master/Voices/gen_fest.wav
The given WAV file I even downloaded and inspected visualized the wave form but it just wouldn't play.
Same problem in Chrome viewing RAW link (or via download link, which is the same difference).

It's an interesting mystery I would likely investigate further, but the Festival site worked fine so I assume it's from that.

**Comment by MasterJames** [comment]
Okay I got it playing with Windows Media Player, VLC Player, QuickTime but Audition and Chrome did not.  Windows 7 File Browser did not provide an inspector either. IE did play it fine. Does not play on Android either, maybe it's a Github RAW WAV issue.
Might be worth converting to MP3, but I guess it's okay for most people. It's pretty bad compared to what's possible today, that/this one sounds like 1st gen Hawkins etc.

**Comment by mramshaw** [comment]
Hey James, sorry I only just spotted this issue (got my free Dot today so I added a note here).

I wanted something that sounded old & horrible (like the VPL). Pretty sure I downloaded it from:

    http://www.cstr.ed.ac.uk/projects/festival/morevoices.html

The RP (received pronunciation, as in _proper English_) voices are pretty good.

I've had good success with Chrome (windows) and chromium (linux).

I've certainly had my issues with sound codecs - MP3 (as you say) is probably the way to go.

[Make sure to ask about codecs next time we meet, it's quite a big subject.]

But even MP3 can be pretty problematic. I always thought WAV was RAW & standard, but maybe not.
